include/ignition/gui/GuiEvents.hh

```diff
@@ -27,6 +27,7 @@
 #include <ignition/common/MouseEvent.hh>
 #include <ignition/math/Vector2.hh>
 #include <ignition/math/Vector3.hh>
+#include <ignition/msgs.hh>
```

Comment:
It's preferable to include just the message that's needed, this helps with compile times.

Reply:
Done in 8e7cb8f